### PR TITLE
Add format() function

### PR DESCRIPTION
**Pull request by StephenWeatherford**
Partial fix for #190

### CODE REVIEW

**Comment by nturinski** [comment]
Your PR fixes itself 😆

File: assets/ExpressionMetadata.json [hunk 1]
```diff
@@ -163,6 +163,13 @@
             "minimumArguments": 1,
             "maximumArguments": 1
         },
```
Comment (EricJizbaMSFT) [inline assets/ExpressionMetadata.json]: I assume you didn't mean to add all the `LanguageServerBin` files...

**Comment by StephenWeatherford** [comment]
> Your PR fixes itself 😆

When you get more experienced, you'll be able to do that as well...

File: assets/ExpressionMetadata.json (hunk 1 above)
Comment (StephenWeatherford) [reply assets/ExpressionMetadata.json]: Fixed

**Pull request merged**